ci/pipeline.yml

```diff
@@ -59,18 +72,64 @@ jobs:
       - put: edge-image
         params:
           image: image/image.tar
+  
+  - name: release
+    plan:
+    - in_parallel:
+      - get: repo
+        trigger: true
+        passed: [ build-edge-image ]
+      - get: edge-image
+        trigger: true
+        passed: [ build-edge-image ]
+        params:
+          format: oci
+      - get: meta-release
+      - get: version
+      - get: charts-repo
+    - task: prep-release
+      config:
+        platform: linux
+        image_resource: #@ release_task_image_config()
+        inputs:
+        - name: repo
+        - name: edge-image
+        - name: meta-release
+        - name: version
+        - name: charts-repo
+        outputs:
+        - name: version
+        - name: artifacts
+        params:
+          CHART: admin-panel
+        run:
+          path: meta-release/ci/tasks/prep-release-src.sh
+    - put: edge-image
```

Comment:
this should not go out to `edge-image` - perhaps `versioned-image` or something like that.

Reply:
yeah, agreed, I have added it!